### PR TITLE
Add note to check docsite on FQCNs after moving modules/plugins to collection

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Add note to check docsite on FQCNs after moving modules/plugins to collection

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`collection_requirements.rst`

### CODE REVIEW

File: collection_requirements.rst [hunk 1]
```diff
@@ -186,6 +186,7 @@ Also:
 
 * change M(), examples, seealso, extended_documentation_fragments to use actual FQCNs (in moved content and in other collections that have references to the content)
 * move all related issues / pull requests / wiki pages
+* look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using ``grep`` command-line utility) to check if there are examples using the moved modules / plugins to actualize their FQCNs
```
Comment (felixfontein) [inline collection_requirements.rst:189]:
```suggestion
* look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using the ``grep`` command-line utility) to check if there are examples using the moved modules / plugins to update their FQCNs
```

Comment (Andersson007) [reply collection_requirements.rst:189]: in my language `actualize` == `update`. Update also works here

Comment (felixfontein) [reply collection_requirements.rst]: In mine too, but not in English :)

Comment (felixfontein) [reply collection_requirements.rst]: https://en.wiktionary.org/wiki/actualize

Comment (Andersson007) [reply collection_requirements.rst]: cool, maybe it came from german originally, thanks for clarifying:)

**Comment by Andersson007** [comment]
cc @gundalow

**Review by gundalow — APPROVED** [review]
Thanks

**Pull request merged**